vbfnlo.spec

```diff
@@ -0,0 +1,24 @@
+### RPM external vbfnlo 3.0.0beta5
+Source: git+https://github.com/vbfnlo/%{n}.git?obj=master/v%{realversion}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
+
+BuildRequires: autotools
+
+%prep
+%setup -q -n %{n}-%{realversion}
+
+%build
+CXX="$(which g++) -std=c++11"
+CC="$(which gcc)"
+FC="$(which gfortran)"
+
```

Comment:
@Andrej-CMS , as mentioned https://github.com/vbfnlo/vbfnlo/blob/master/README.md , you need to run `autoreconf -vi` before running `configure`